### ISSUE TITLE
Does framework validate the JWT token before creating PrincipalClaim ?

### ISSUE THREAD

**Issue by vpnchoudhary** [Azure/Azure-Functions#1275]
Please describe the documentation request or issue.  Be as specific as possible.
I have easy auth setup on my Azure function. Does the JWT token is verified by the framework for validity before creating PrincipalClaim ?

**Comment by ColbyTresness** [comment]
@mattchenderson

**Comment by ConnorMcMahon** [comment]
Yes, the Functions Runtime populates the Claims Principal with the identity from EasyAuth. EasyAuth handles the validation of the JWT.
